### PR TITLE
Draft: 42 display eurosign

### PR DESCRIPTION
**Pull request by Kangaroo6942**
added the ability to add a font with eurosign. Strangely all unknown characters default to the eurosign

### CODE REVIEW

**Review by lchristus — CHANGES_REQUESTED** [review]
very nice just slight mistake. Also is it possible to install the font by default?

**Comment by Kangaroo6942** [comment]
Since it is currently a problem that all special characters default to the Eurosign I wouldn't want to install it per default.
I also looked for a possibility to add the font when installing the widget, there probably is a possibiliy to provide a script for installation but I see that more as a concern of the initial setup/settings ticket.
If that feature is still desired, I have to go ask the teaching chair tomorrow.

File: src/MensaWidget/MWFontLoader.class.st
[the remark this answers is not in the document]
Comment (Kangaroo6942) [reply src/MensaWidget/MWFontLoader.class.st:1]: done

File: src/MensaWidget/MWFontLoader.class.st [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	#'squeak_changestamp' : 'Max 6/11/2024 18:58'
+}
+MWFontLoader class >> loadFallbackFont [
+	
+	| fontReader font |
+	
+	fontReader := TTFontReader readFrom: self fontAsset readStream.
+	font := TTCFont newTextStyleFromTT: fontReader first.
+	TextStyle setDefaultFallback: font.	
+	TextStyle installDefaultFallbackTextStyle
```
[the remark this answers is not in the document]
Comment (Kangaroo6942) [reply src/MensaWidget/MWFontLoader.class.st]: done

**Review by lchristus — APPROVED** [review]
i would talk to the client how these issues should be handled

**Review by NilsKasper — COMMENTED** [review]
seems good

**Comment by lchristus** [comment]
This is not a requirement anymore therefor this will be closed

**Pull request closed without merging**